src/compositor/shells/xdg_v6.rs

```diff
@@ -43,6 +64,49 @@ impl XdgV6ShellHandler for XdgV6 {
         }).unwrap();
     }
 
+    fn on_commit(&mut self,
+                 compositor: CompositorHandle,
+                 _: SurfaceHandle,
+                 shell_surface: XdgV6ShellSurfaceHandle) {
+        let configure_serial = {
+            with_handles!([(shell_surface: {shell_surface.clone()})] => {
+                shell_surface.configure_serial()
+            }).unwrap()
+        };
+
+        let surface = shell_surface.into();
+        with_handles!([(compositor: {compositor})] => {
+            let server: &mut Server = compositor.into();
+            let Server { ref mut seat,
+                         ref mut views,
+                         ref mut cursor,
+                         .. } = *server;
+            let idx = views.iter().position(|view| view.shell == surface);
+            if let Some(idx) = idx {
+                let view = views.get(idx).unwrap().clone();
+                if let Some(move_resize) = view.pending_move_resize.get() {
+                    if move_resize.serial > 0 && move_resize.serial >= configure_serial {
```

Comment:
Can/should a serial ever by < 0? Should we switch this to e.g. a `u64` or something?

Reply:
configure_serial is uint32 in wlroots.

That check isn't required though since we already check for 0 serial earlier. That's the case when no configure was sent because the properties are the same as we are trying to set.